core/src/main/java/org/jdbi/v3/core/statement/SqlStatement.java

```diff
@@ -1811,6 +1803,19 @@ PreparedStatement createStatement(final StatementContext ctx, ParsedSql parsedSq
         return handle.getStatementBuilder().create(handle.getConnection(), parsedSql.getSql(), ctx);
     }
 
+    ParsedSql templateSql() {
```

Comment:
This is a minor nit since it's not public API, but this method is rendering a template and parsing SQL, not just templating. And the return type is `ParsedSql`. If we want a terse name, `parseSql` is probably better IMO.